spacepy/toolbox/__init__.py

```diff
@@ -1522,26 +1527,22 @@ def windowMean(data, time=[], winsize=0, overlap=0, st_time=None, op=np.mean):
     else:
         if len(data) != len(time):
             raise ValueError('windowmean error: data and time must have same length')
-        #First check if datetime objects
-        try:
-            assert type(winsize) == datetime.timedelta
-            assert type(overlap) == datetime.timedelta
-        except AssertionError:
-            raise TypeError('windowmean error: winsize/overlap must be timedeltas if a time array is supplied.')
         pts = False #force time-based averaging
-        if (type(time[0]) != datetime.datetime):
+        #First check if datetime objects
+        if type(time[0]) == datetime.datetime:
```

Comment:
Can `type(time[0]) == datetime.datetime` be replaced with `isinstance(time[0], datetime.datetime)`? Same with next few lines.

Reply:
That's more "it was like that before and minimum change to make it work" but yeah, makes sense to fix that.